src/librbd/Utils.cc

```diff
@@ -194,5 +194,11 @@ SnapContext get_snap_context(
   return snapc;
 }
 
+uint64_t reserve_async_request_id() {
+  static std::atomic<uint64_t> async_request_seq;
```

Comment:
Nit: does this correctly initialize the sequence to zero? [1] seems to hint that the wrapped object is not initialized.

[1] https://en.cppreference.com/w/cpp/atomic/atomic/atomic

Reply:
I used from C times that all static variables are always initialized to 0 if not explicitly initialized. Ok, I will add the explicit initialization so it does not rise the question.